From 82a4fd7b5aa7c8e10a9f6c0d23d987aa0a1e7f45 Mon Sep 17 00:00:00 2001
From: Rick Bouma <r.bouma@disrex.nl>
Date: Wed, 26 Mar 2025 10:16:23 +0100
Subject: [PATCH] Move exception.phtml to base to support backend Magewire

This change was added later by Rick Bouma (r.bouma@disrex.nl)
to ensure Magewire exceptions work in both frontend and backend.

---
 src/view/base/templates/component/exception.phtml | 0
 .../templates/component/exception.phtml           | 0
 2 files changed, 0 insertions(+), 0 deletions(-)
 rename src/view/{frontend => base}/templates/component/exception.phtml (100%)

diff --git a/src/view/frontend/templates/component/exception.phtml b/src/view/base/templates/component/exception.phtml
similarity index 100%
rename from src/view/frontend/templates/component/exception.phtml
rename to src/view/base/templates/component/exception.phtml
